modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1381,6 +1381,12 @@ def stack(self, level, dropna):
             pandas.to_numeric(df.squeeze(axis=1), *args, **kwargs)
         )
     )
+    to_timedelta = Map.register(
+        lambda s, *args, **kwargs: pandas.to_timedelta(
+            s.squeeze(1), *args, **kwargs
```

Comment:
```suggestion
            s.squeeze(axis=1), *args, **kwargs
```
is that right? I would prefer using named arguments for better readability

Reply:
Fixed.